install_config/persistent_storage/persistent_storage_iscsi.adoc

```diff
@@ -84,3 +83,33 @@ namespace. Any attempt to access a persistent volume across a namespace causes
 the pod to fail.
 
 Each iSCSI LUN must be accessible by all nodes in the cluster.
+
+[[iscsi-multipath]]
+
+=== iSCSI Multipathing
+For iSCSI based storage, you can configure multiple paths by using same IQN for more than one target portal IP addresses. Multipathing ensures access to the persistent volume when one or more of the components in a path fail.
```

Comment:
s/iSCSI based/iSCSI-based

s/by using same/by using the same

s/IP addresses/IP address